modal.js

```diff
@@ -0,0 +1,103 @@
+const card1 = {
+    title: 'Keeping track of hundreds  of components website',
+    techList: [
+        'html',
+        'Bootstrap',
+        'Ruby on rails'
+    ],
+    description: `'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+      standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to
+      make a type specimen book. It has survived not only five centuries, but also the leap into electronic
+      typesetting, remaining essent'`,
+    link: [
+      'https://LeslieAine.github.io/PortfolioSetup/',
+      'https://github.com/LeslieAine/',
+    ],
+    image: 'images/card1.png',
+  };
+  
+  const card2 = {
+    title: 'Multi Post Stories',
+    techList: [
+        'html',
+        'Bootstrap',
+        'Ruby on rails'
+    ],
+    description: `'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+      standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to
+      make a type specimen book. It has survived not only five centuries, but also the leap into electronic
+      typesetting, remaining essent'`,
+    link: [
+        'https://LeslieAine.github.io/PortfolioSetup/',
+        'https://github.com/LeslieAine/',
+    ],
+    image: 'images/card2.png',
+  };
+  
+  const card3 = {
+    title: 'Multi Post Stories',
+    techList: [
+        'html',
+        'Bootstrap',
+        'Ruby on rails'
+    ],
+    description: `'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+      standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to
+      make a type specimen book. It has survived not only five centuries, but also the leap into electronic
+      typesetting, remaining essent'`,
+    link: [
+        'https://LeslieAine.github.io/PortfolioSetup/',
+        'https://github.com/LeslieAine/',
+    ],
+    image: 'images/card2.png',
+  };
+  
+  const card4 = {
+    title: 'Multi Post Stories',
+    techList: [
+        'html',
+        'Bootstrap',
+        'Ruby on rails'
+    ],
+    description: `'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+      standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to
+      make a type specimen book. It has survived not only five centuries, but also the leap into electronic
+      typesetting, remaining essent'`,
+    link: [
+        'https://LeslieAine.github.io/PortfolioSetup/',
+        'https://github.com/LeslieAine/',
+    ],
+    image: 'images/card2.png',
+  };
+  
+  const projects = [card1, card2, card3, card4];
```

Comment:
- [ ] If your pop-up cards don't have any id, how do you assign each card to its specific popup window data? as a test, try to change images or title for each pop-up window and try to figure out how it is not fair to not use ids, please make sure if the user clicks on the see project of the 1st card, it will not take a random or same pop up for all card  and display it
- Take an example from this code [here](https://github.com/joskalenda/portfolio/pull/8/files#diff-e891d369d4489d49744f08f34a2095d795bbffcc5bf3fb9d7d1904de543f0f27), but do not copy as it is forbidden by microverse, see the way this student created the array and rendered the data dynamically, hope it helps ❤️ 

- [ ] Let's touch on the design a bit, am gonna drop 2 images and a screen record below to show you what is to be changed

YOUR DESIGN POPUP

<img width="701" alt="2" src="https://user-images.githubusercontent.com/77843050/210662880-120b77ac-7086-4093-a289-bf8accb49676.png">

- RECORD

![5](https://user-images.githubusercontent.com/77843050/210663368-c14dd7fa-9cdd-4be1-8947-e83480a98296.gif)


- Please make the update to stick as much as possible to this correct design below, we need a conformed copy of it, please make sure you check every aspect and make the modification @LeslieAine 

FIGMA DESIGN POPUP

<img width="771" alt="1" src="https://user-images.githubusercontent.com/77843050/210662966-dd8eec73-4f26-41a8-9d1a-c7bcbc9c5760.png">